src/topp/GNPSExport.cpp

```diff
@@ -42,24 +43,91 @@
+and corresponding feature quantification table (.TXT format) that contains the LC-MS peak area intensity.
+
+Requirements:
+	- The IDMapper has to be ran on the featureXML files, in order to associate MS2 scan(s) (peptide annotation) with each
+	features. These peptide annotations are used by the GNPSExport.
+	- The FileFilter has to be ran on the consensusXML file, prior to the GNPSExport, in order to remove consensusElements
+	without MS2 scans (peptide annotation).
+
+Parameters:
+	- Binning (ms2_bin_size): Defines the binning width of fragment ions during the merging of eligible MS/MS spectra.
+	- Cosine Score Treshold (merged_spectra:cos_similarity): Defines the necessary pairwise cosine similarity with the highest precursor intensity MS/MS scan.
+
+  - Output Type (output_type):
+Options for outputing GNPSExport spectral processing are:
+    -# [RECOMMENDED] merged_spectra
+      For each consensusElement, the GNPSExport will merge all the eligible MS/MS scans into one representative consensus MS/MS spectrum.
+      Eligible MS/MS scans have a pairwise cosine similarity with the MS/MS scan of highest precursor intensity above the Cosine Similarity Treshold.
+	    The fragment ions of merged MS/MS scans are binned in m/z (or Da) range defined by the Binning width parameter.    
+      .
+	  -# Most intense: most_intense - For each consensusElement, the GNPSExport will output the most intense MS/MS scan (with the highest precursor ion intensity) as consensus MS/MS spectrum.
+      .    
+
+Note that mass accuracy and the retention time window for the pairing between MS/MS scans and a LC-MS feature
+or consensusElement is defined at the IDMapper tool step.
+
+A representative OpenMS-GNPS workflow would sequencially use these OpenMS TOPP tools:
   1. Input mzML files
-  2. Run the FeatureFinderMetabo tool
-  3. Run the IDMapper tool
-  4. Run the MapAlignerPoseClustering tool
-  5. Run the FeautureLinkerUnlabeledKD tool
-  6. Run the FileConverter tool (output FeatureXML format)
-  7. Run the MetaboliteAdductDecharger
-  8. Run the GNPSExport to export an .MGF
-  9. Upload the .MGF file on http://gnps.ucsd.edu and follow the instructions here:
+  2. Run the FeatureFinderMetabo tool on the mzML files.
+  3. Run the IDMapper tool on the featureXML and mzML files.
+  4. Run the MapAlignerPoseClustering tool on the featureXML files.
+  5. Run the MetaboliteAdductDecharger on the featureXML files.
+  6. Run the FeatureLinkerUnlabeledKD tool or FeatureLinkerUnlabeledQT, on the featureXML files and output a consensusXML file.
+  8. Run the FileFilter on the consensusXML file to keep only consensusElements with at least MS/MS scan (peptide identification).
+  9. Run the GNPSExport on the "filtered consensusXML file" to export an .MGF file.
+  10. Run the TextExport on the "filtered consensusXML file" to export an .TXT file.
```

Comment:
do you mean `TextExporter`?

Reply:
you can add hyperlinks to the above TOPP tools by using e.g. `@ref TOPP_FileFilter` instead of just `FileFilter`